accounts/addproofer.php

```diff
@@ -80,15 +80,18 @@
             setcookie("http_referer", '', time() - 1);
 
             // Page shown when account is successfully created
-            $header = sprintf(_("User %s Registered Successfully"), $username);
-            output_header($header);
+            $title = _("Registration complete");
+            output_header($title);
+            echo "<h1>$title</h1>";
 
             // Send them an activation e-mail
             maybe_activate_mail($email, $real_name, $register->id, $username, $intlang);
 
+            echo "<p>";
             echo sprintf(
                _("User %s registered successfully. Please check the e-mail being sent to you for further information about activating your account. This extra step is taken so that no-one can register you to the site without your knowledge."),
                $username);
```

Comment:
should we html_safe this since it includes untrusted user input (username)? Though I'm pretty sure our regexp on username's prevents any html characters.

Reply:
The regex prevents invalid characters, but this is easy and we should do it anyway.